proto/ignition/msgs/statistic.proto

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+syntax = "proto3";
+package ignition.msgs;
+option java_package = "com.ignition.msgs";
+option java_outer_classname = "StatisticProtos";
+
+/// \ingroup ignition.msgs
+/// \interface Statistic
+/// \brief A message that contains statistics data.
+
+import "ignition/msgs/header.proto";
+
+message Statistic
+{
+  enum DataType
+  {
+    UNITIALIZED = 0;
+    AVERAGE      = 1;
+    MINIMUM      = 2;
+    MAXIMUM      = 3;
+    STDDEV       = 4;
+    SAMPLE_COUNT = 5;
```

Comment:
in [ign-math SignalStats.hh](https://github.com/ignitionrobotics/ign-math/blob/ign-math6/include/ignition/math/SignalStats.hh), we also have `RootMeanSquare`, `MaxAbsoluteValue`, and `Variance` in case we want to add those to the enum as well

Reply:
Is `MaxAbsoluteValue` the same as `max(abs(MINIMUM), abs(MAXIMUM))`? If so, would you prefer to call out a separate field for `MaxAbsoluteValue`, or leave it as `MNUMUM` and `MAXIMUM`?

Reply:
yes, `MaxAbsoluteValue` is the same as `max(abs(MINIMUM), abs(MAXIMUM))`, so I don't think you would need all 3 in a `StatisticsGroup`. I was assuming that a `StatisticsGroup` was not required to contain every single `DataType`, and thought that it would be useful to add more types to this `enum`. Do we need this `enum` to be carefully curated?

Reply:
That's a good point, you could have the situation where you just want to store/transmit the `maxabsvalue`. I'll add it in.

Reply:
Done: 530ca4f9a949db6c87c5a170372dac2a5bee11ed